docs/system_overview/configuration.md

```diff
@@ -56,20 +68,54 @@ This file contains the following top level entries:
 
   - `default_resource_name` (optional) - The name of the initial resource on server installation
 
-  - `environment_variables` (required) - Contains a set of key/value properties of the form VARIABLE=VALUE such as "ORACLE_HOME=/full/path" from the server's environment.  Can be empty.
+  - `environment_variables` (required) - Contains a set of key/value properties of the form VARIABLE=VALUE such as "ORACLE_HOME=/full/path" from the server's environment.  Can be empty
 
   - `federation` (required) - Contains an array of objects which each contain the parameters necessary for federating with another grid.  The array can be empty, but if an object exists, it must contain the following properties:
-    - `catalog_provider_hosts` (required) -  An array of hostnames of the catalog service providers in the federated zone.
-    - `negotiation_key` (required) - The 32-byte encryption key of the federated zone.
-    - `zone_key` (required) - The shared authentication secret of the federated zone.
-    - `zone_name` (required) -  The name of the federated zone.
 
-  - `catalog_provider_hosts` (required) - An array of fully qualified domain names of this Zone's catalog service provider
+      - `catalog_provider_hosts` (required) -  An array of hostnames of the catalog service providers in the federated zone
 
-  - `catalog_service_role` (required) - The role of this server, either 'provider' or 'consumer'
+      - `negotiation_key` (required) - The 32-byte encryption key of the federated zone
+
+      - `zone_key` (required) - The shared authentication secret of the federated zone
+
+      - `zone_name` (required) -  The name of the federated zone
+
+  - `host_access_control` (required) - See [Host Access Control](#host-access-control) for more details
+
+      - `access_entries` (required)
+
+  - `host_resolution` (required) - See [Host Resolution](#host-resolution) for more details
+
+      - `host_entries` (required)
 
   - `kerberos_name` (optional) - Kerberos Distinguished Name for KRB and GSI authentication
```

Comment:
Should this be listed here?

The name doesn't appear in the Kerberos configuration section. Given that there's a whole section on how to configure Kerberos, it seems reasonable to remove this.

Reply:
I think that's a correct assessment - it does feel weird to see it there now.

Reply:
removed